hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -261,7 +262,11 @@ public void testCreateTableWithOwner() throws Exception {
     String location = temp.newFolder("tbl").toString();
     String owner = "some_owner";
     ImmutableMap<String, String> properties =
-        ImmutableMap.of(TableProperties.HMS_TABLE_OWNER, owner);
+        ImmutableMap.of(
+            TableProperties.HMS_TABLE_OWNER,
+            owner,
+            TableProperties.HMS_TABLE_OWNER_TYPE,
```

Comment:
I'd also keep the test coverage when there is no HMS_TABLE_OWNER_TYPE is set.

Reply:
Yes, good idea.